platform/ios/export/export_plugin.cpp

```diff
@@ -573,11 +574,14 @@ Error EditorExportPlatformIOS::_export_icons(const Ref<EditorExportPreset> &p_pr
 				ERR_PRINT("Invalid icon (" + String(info.preset_key) + "): '" + icon_path + "'.");
 				return ERR_UNCONFIGURED;
 			}
+			if (info.force_opaque && img->detect_alpha() != Image::ALPHA_NONE) {
+				ERR_PRINT("Icon (" + String(info.preset_key) + ") must be opaque.");
+				return ERR_UNCONFIGURED;
+			}
 			img->resize(side_size, side_size);
 			err = img->save_png(p_iconset_dir + info.export_name);
 			if (err) {
-				String err_str = String("Failed to export icon(" + String(info.preset_key) + "): '" + icon_path + "'.");
-				ERR_PRINT(err_str.utf8().get_data());
+				ERR_PRINT("Failed to export icon(" + String(info.preset_key) + "): '" + icon_path + "'.");
```

Comment:
This should use `add_message` to display a message in the UI as well (seems like I missed a bunch of `ERR_PRINTS` when adding it, all of them should be changed this way):

```suggestion
				add_message(EXPORT_MESSAGE_ERROR, TTR("Export Icons"), vformat(TTR("Failed to export icon(%s): '%s'."), info.preset_key, icon_path));
```

Reply:
Is https://github.com/godotengine/godot/pull/64632 still a good change, or should it be closed?

Reply:
I guess it's still a good change, since some stuff like verbose output from the external apps is not in the UI.